poetry.lock

```diff
@@ -1606,6 +1608,14 @@ sanic = ["sanic (>=0.8)"]
 sqlalchemy = ["sqlalchemy (>=1.2)"]
 tornado = ["tornado (>=5)"]
 
+[[package]]
+name = "sgmllib3k"
```

Comment:
Any idea what's going on here? The package's description is dire. Is this part of feedparser? I hope not.

Reply:
It is a `feedparser 6.0.8` dependency:

https://github.com/kurtmckee/feedparser/blob/develop/pyproject.toml#L21-L23

Reply:
always one q.  yes its dire sounding but... that's never stopped us before right?